command/app.go

```diff
@@ -64,6 +64,10 @@ var app = &cli.App{
 			Name:  "stat",
 			Usage: "collect statistics of program execution and display it at the end",
 		},
+		&cli.BoolFlag{
+			Name:  "no-sign-request",
+			Usage: "do not sign requests",
```

Comment:
should we indicate that the credentials won't be loaded?

Reply:
updated both help text and changelog